modules/hosted-cluster-etcd-status.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assembly:
+//
+// * hcp-backup-restore-dr.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-cluster-etcd-status_{context}"]
+= Checking the status of a hosted cluster
+
+To check the status of your hosted cluster, complete the following steps.
+
+.Procedure
+
+. Enter the running etcd pod that you want to check by entering the following command:
++
+[source,terminal]
+----
+$ oc rsh etcd-0
+----
+
+. Set up the etcdctl environment by entering the following commands:
++
+[source,terminal]
+----
+export ETCDCTL_API=3
+export ETCDCTL_CACERT=/etc/etcd/tls/etcd-ca/ca.crt
+export ETCDCTL_CERT=/etc/etcd/tls/client/etcd-client.crt
+export ETCDCTL_KEY=/etc/etcd/tls/client/etcd-client.key
+export ETCDCTL_ENDPOINTS=https://etcd-client:2379
+----
```

Comment:
Do we need to add the prompt for these commands (`sh-4.4#`)? 
Should these commands be in separate code blocks, [per the guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output)?

Reply:
Yep. I just updated those commands and the one in step 3.

Reply:
I believe the commands here should be `sh-4.4# export ..` The pod itself doesn't have the `oc` client installed.
This is what I am getting:
```
sh-4.4# oc export ETCDCTL_CACERT=/etc/etcd/tls/etcd-ca/ca.crt
sh: oc: command not found
sh-4.4# export ETCDCTL_CACERT=/etc/etcd/tls/etcd-ca/ca.crt
sh-4.4# 
```